Objective-C/CBLDatabase.mm

```diff
@@ -1008,7 +1008,7 @@ - (BOOL) saveDocument: (CBLDocument*)document
                 
                 C4Error err;
                 CBLStringBytes bDocID(document.id);
-                curDoc = c4doc_get(_c4db, bDocID, true, &err);
+                curDoc = c4db_getDoc(_c4db, bDocID, true, kDocGetCurrentRev, &err);
```

Comment:
I searched through the three linked docs but couldn't find any reference to this function.  Could you point it out?

Reply:
Apparently that function is not in the three version vector docs. The information can be found in the API doc itself. I will include this in the PR's description.

https://github.com/couchbase/couchbase-lite-core/blob/master/C/include/c4Document.h#L101-L122